public/static/locales/eng/cookies.md

```diff
@@ -1,26 +1,25 @@
-# Cookies
+# Cookie policy
 
 NOTE: Translation efforts are in progress. Until then, please direct any questions to samromur@ru.is.
 
-Þessi vefsíða notar vafrakökur (e. cookies) til að bæta upplifun þína á vefsíðunni.
+This website uses cookies to improve your experience on the website.
 
-## 1. Hvað er vafrakaka?
+## 1. What is a cookie?
 
-Vafrakökur (e. cookies) eru nokkurs konar fótspor sem eru dulkóðaðir textastrengi sem vefsíða geymir á tölvu notandans í formi textaskráar. Þær gerðir af kökum sem við notum eru „lotukökur“ (e. session cookies) og „viðvarandi kökur“ (e. persistent cookies). Lotukökur er tímabundnar og er sjálfkrafa eytt þegar þú yfirgefur vefsíðuna okkar. Viðvarandi kökur eru áfram í tölvunni þinni þar til þú eyðir þeim.
-Vefgeymsla vafra (e. web storage) gerir vefsvæðum kleift að geyma gögn í tölvu notanda líkt og vafrakökur nema eru einungis lesanlegar notanda megin og er stærri en vafrakaka (e. cookie). Í vefgeymslu vafrans notandans nýtir vefurinn „staðbundina geymslu“ (e. local storage) og „lotu geymslu“ (e. session storage). Gögn í lotu geymslu vafrans eru tímabundin og er sjálfkrafa eytt þegar þú yfirgefur vefsíðuna. Gögn í staðbundinni geymslu vafrans eru áfram í tölvunni þinni þar til þú eyðir þeim.
+Cookies are encrypted text strings that a website stores on the user's computer in the form of a text file. The types of cookies we use are "session cookies" and "persistent cookies". Session cookies are temporary and are automatically deleted when you leave our website. Persistent cookies remain on your computer until you delete them.
 
-Án notkunnar á vafrakökum og vefgeymslu vafra gæti upplifun þín á síðunni versnað og vefsíðan ekki virkað eins og til var ætlast.
+Without the use of cookies and browser web storage, your experience on the site may deteriorate and the site may not function as intended.
 
-## 2. Tilgangur með notkun vafrakaka og vefgeymslu vafra.
+## 2. Purpose of using cookies and browser web storage.
 
-Vefsíðan notar vafrakökur til að mæla notkun á ýmsum undirsíðum innan vefsíðunnar, en það hjálpar okkur að meta hvað þarf að bæta, til þess notum við vafrakökur fyrir Google Analyctics (nafn: ga). Með því getum við séð notkunarmynstur á síðunni yfir heildina í stað þess að sjá notkun einstaka notanda. Við notum upplýsingarnar til að greina umferð á vefsíðunni en ekki til að skoða persónugreinanlegar upplýsingar.
+The website uses cookies to measure the use of various sub-pages within the website, but it helps us to assess what needs to be improved, for this we use cookies for Google Analytics. This allows us to see the usage patterns of the site as a whole instead of seeing the usage of individual users. We use the information to analyse website traffic and not to view personally identifiable information.
```

Comment:
missing (name: ga)

Reply:
We are actually not using google analytics so I will remove that part.

Reply:
in both the icelandic and english portion?